### PR TITLE
Use I3 for xiwi instead of ratpoison

### LINKED ISSUE #1483: Xiwi and Java GUI applications
Neither Eclipse, nor JetBrains IDEA are working in xiwi window (launched as `xiwi idea` inside chroot). The window is blank, but when I click on some hidden controls, some popups or drop downs are working.

I'm using trusty release.

### PR DESCRIPTION
**Pull request by dnschneid**
Improves floating window behavior and compatibility with software that requires a reparenting window manager.

Window title monitoring is also more efficient.

Fixes #1483

Colors are also set to look nice with stock Chromium OS.

### CODE REVIEW

**Comment by fczuardi** [comment]
I haven't tested this version yet, will ```xiwi i3``` still work with this?

**Comment by dnschneid** [comment]
What are you trying to do?  If you just want to launch an i3 environment, you're better off configuring your `.xinitrc` and using xinit like on a normal system.

The I3 config used by xiwi will not touch your `.i3/config`.

**Comment by DennisLfromGA** [comment]
I'm running Popcorn-Time that kicks-off in a background process using the i3xiwi branch!

**Comment by dnschneid** [comment]
Merge first, ask forgiveness/apply @drinkcat's suggestions later :shipit:

File: chroot-etc/xiwi.conf [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# i3 config file (v4)
+# Copyright (c) 2014 The crouton Authors. All rights reserved.
```
Comment (drinkcat) [inline chroot-etc/xiwi.conf]: 2015 ,-)

**Comment by dnschneid** [comment]
I think you can put i3 commands inside of the .xiwirc file, so you can effectively already do that.

**Pull request merged**